src/api/app/controllers/webui/request_controller.rb

```diff
@@ -75,8 +75,6 @@ def modify_review
   def show
     diff_limit = params[:full_diff] ? 0 : nil
     @req = @bs_request.webui_infos(filelimit: diff_limit, tarlimit: diff_limit, diff_to_superseded: @diff_to_superseded)
-    @id = @req['id']
-    @number = @req['number']
     @state = @req['state'].to_s
```

Comment:
what about getting rid of `@state`, ` @accept_at`,  `@superseded_by`,  `@superseding` as well? :confused:

Reply:
it is working in progress.. please calm down :)

Reply:
how I am suppose to know this is a WIP? Please do not send PR that are not ready to be review... and if you do it, at least write somewhere that it is a WIP... :wink: